test/assignment3/Assignment3Test.java

```diff
@@ -0,0 +1,81 @@
+package assignment3;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Assignment3Test {
+    
+    private static final int HEIGHT = 2;
+    private static final int WIDTH = 3;
+    private static final char[][] GRID = new char[HEIGHT][WIDTH];
+    
+    @BeforeClass
+    public static void setUpClass() {
+        Assignment3Test.GRID[0][0] = 'a';
+        Assignment3Test.GRID[0][1] = 'a';
+        Assignment3Test.GRID[0][2] = 'r';
+        
+        Assignment3Test.GRID[1][0] = 't';
+        Assignment3Test.GRID[1][1] = 'c';
+        Assignment3Test.GRID[1][2] = 'd';
+    }

```

Comment:
You could also write `GRID = {{'a','a','r'},{'t','c','d'}};` without `Assignment3Test.`. You don't even need the `setUpClass`method, while declaring `GRID`, you could just say: 
`private final char[][] GRID = {{'a','a','r'},{'t','c','d'}};`